pig/src/main/java/org/jboss/pnc/bacon/pig/impl/repo/RepoManager.java

```diff
@@ -388,8 +405,65 @@ private void addExtraFiles(File m2Repo) {
         }
     }
 
+    /**
+     * Generates the offline manifest text file, which contains all the dependencies, including third party, non-RH
+     * dependencies. First, it adds the build artifacts for all the builds. Then, it executes a query to PNC, requesting
+     * the third party dependencies for each build. Finally, the method generates a file, containing version paths and
+     * checksums, where we can get it, for all build and dependency attributes. The offline manifest file in combination
+     * with the offliner tool (https://repo1.maven.org/maven2/com/redhat/red/offliner/offliner) are used to download all
+     * the attributes to a local maven repository.
+     */
+
     @Override
```

Comment:
@akoufoudakis Move the `close()` method to the end, and move your method to match the javadoc. The javadoc is still on the wrong method it looks like.